node-operators/networks/run-a-node/systemd.md

```diff
@@ -334,6 +334,37 @@ For an overview of the flags used in the following start-up commands, plus addit
 
 ### Collator {: #collator }
 
+Beginning with v0.39.0, Moonbeam collator nodes no longer generate session keys automatically on start-up. Run the following command to generate and store on disk the session keys that will be referenced in the start-up command:
```

Comment:
"Also important to leave a not that this has to be done for people running new nodes, not people with existing ones" - Alberto

Reply:
Added language about this being for new nodes.